lib/groonga-query-log/command/run-regression-test.rb

```diff
@@ -56,6 +60,17 @@ def initialize
         @target_command_names = ServerVerifier::Options.new.target_command_names
 
         @read_timeout = Groonga::Client::Default::READ_TIMEOUT
+
+        @mail_notification = false
+        @mail_subject_on_succeeded = "Succeeded"
+        @mail_subject_on_failure = "Failed"
+        @mail_from = "groonga-query-log@example.com"
```

Comment:
`"groonga-query-log@#{Socket.hostname}` is better as the default from. Because `example.com` doesn't exist.